src/selectors/BlogSelectors.js

```diff
@@ -0,0 +1,9 @@
+export const getBlog = state => state;
+
```

Comment:
You'll also want to get a post by index. And I would name the above selector getBlogs since you're getting all of them. :)